luna/featurevis/transformations.py

```diff
@@ -3,53 +3,118 @@
 """
 import random
 import tensorflow as tf
+import tensorflow_addons as tfa
 
 
-def add_noise(img, noise):
+def noise(img, add_noise):
```

Comment:
Couldn't we remove the second param here? Would we ever want to call this function if we do not want to add noise?

Reply:
Resolved by giving transformations as function.